### PR TITLE
Adds Two New Workflows

### PR DESCRIPTION
**Pull request by kidunot89**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
Migrates to Travis-CI to Github Actions

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** Linux Mint 19.2

**WordPress Version:** 5.3.2

### CODE REVIEW

**Comment by renatonascalves** [comment]
I don't see the PHPCS running here: https://github.com/wp-graphql/wp-graphql-woocommerce/pull/239/checks?check_run_id=456668984

Is it the right place?

**Comment by kidunot89** [comment]
> I don't see the PHPCS running here: https://github.com/wp-graphql/wp-graphql-woocommerce/pull/239/checks?check_run_id=456668984
> 
> Is it the right place?

You're in the right place, just click `Run`.

**Comment by renatonascalves** [comment]
This is what I get. That doesn't seem right.

Screen Shot 2020-02-20 at 11 19 36 AM

**Comment by kidunot89** [comment]
I'm pretty sure that correct, but I'm mess something up just to see if it returns a failure.

**Comment by renatonascalves** [comment]
Since the phpcs file is telling to show the diff [here](https://github.com/wp-graphql/wp-graphql-woocommerce/blob/0603f830ebb375d19cf01dab6a8fbf0cc1a2dfbb/.phpcs.ruleset.xml#L14), we should be getting a few dots of passed tests there....

File: includes/model/class-coupon.php [hunk 1]
```diff
@@ -1,12 +1,5 @@
 <?php
-/**
- * Model - Coupon
- *
- * Resolves coupon crud object model
- *
- * @package WPGraphQL\WooCommerce\Model
- * @since 0.0.1
- */
+
```
Comment (kidunot89) [inline includes/model/class-coupon.php]: For PHPCS failure test

**Comment by kidunot89** [comment]
You may be correct. I'm not getting any changes in the WPCS workflow results despite removing the top doc block in the `includes/model/class-coupon.php`.

**Comment by kidunot89** [comment]
Fixed it, just had to update the **PHPCS** ruleset
image

**Comment by renatonascalves** [comment]
@kidunot89 Ok!

A few suggestions! You should remove that line:

```
vendor/bin/phpcs \
  vendor/bin/phpcs \
  wp-graphql-woocommerce.php \
  access-functions.php \
  class-inflect.php \
  includes/*.php --standard=WordPress
```

The only thing that should be here is:

```
vendor/bin/phpcs
```

The rest should be added to the `phpcs.xml` file.

**Comment by renatonascalves** [comment]
I don't think this is getting all the files. I'd recommend removing inline values and let the `phpcs.xml` do the rest.

Adding the standard, folders to fetch and other values there. Rather than inline.

Screen Shot 2020-02-20 at 1 06 54 PM

**Comment by kidunot89** [comment]
Good suggestion. I just realized that using `includes/*.php` only target the top level php files.

**Comment by renatonascalves** [comment]
@kidunot89 Great! Most likely after you add the folders, we will get lots of PHPCS errors. I'll help as much as I can in fixing them. :)

**Comment by kidunot89** [comment]
@renatonascalves There were a ton, but I got them. :+1:

**Comment by renatonascalves** [comment]
@kidunot89 Recommend adding this:

`phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound`

in the `phpcs.xml`  file. This should be a global override rather than inline.

**Comment by kidunot89** [comment]
Naw, flag is only used on hooks that can't be changed due to be woocomerce hook or adhere to WPGraphQL standards. All others I changed.

**Pull request merged**